resources/js/components/Room/HistoryComponent.vue

```diff
@@ -0,0 +1,420 @@
+              >
+                <i class='fas fa-chart-line'></i>
+              </b-button>
+              <b-button
+                v-b-tooltip.hover
+                :title="$t('meetings.viewMeetingAttendance')"
+                :disabled='meetingsLoading || statsLoading || attendanceLoading'
+                v-if="data.item.attendance && data.item.end != null"
+                variant='primary'
+                @click="loadMeetingAttendance(data.item)"
+              >
+                <i class="fas fa-user-clock"></i>
+              </b-button>
+            </template>
+          </b-table>
+          <b-pagination
+            v-model='meetingsMeta.current_page'
+            :total-rows='meetingsMeta.total'
+            :per-page='meetingsMeta.per_page'
+            aria-controls='meetings-table'
+            @input="$root.$emit('bv::refresh::table', 'meetings-table')"
+            align='center'
+            :disabled='meetingsLoading || meetingsLoadingError'
+          ></b-pagination>
+
+          <div v-if="settings('attendance.enabled') || settings('statistics.meetings.enabled')" id="retentionPeriodInfo">
+            <hr>
+            <b>{{ $t('meetings.retentionPeriod') }}</b><br>
+            <span v-if="settings('statistics.meetings.enabled')">{{ $t('meetings.stats.retentionPeriod', {'days': settings('statistics.meetings.retention_period')}) }}</span><br>
+            <span v-if="settings('attendance.enabled')">{{ $t('meetings.attendance.retentionPeriod', {'days': settings('attendance.retention_period')}) }}</span><br>
+          </div>
+        </div>
+      </div>
+
+      <!-- Statistics modal -->
+      <b-modal :static="modalStatic" size="xl" hide-footer id="statsModal">
+        <template #modal-title>
+          <h5 v-if="statsMeeting">{{ $t('meetings.stats.modalTitle',{room: room.name }) }}
+          <br><small>{{ $date.utc(statsMeeting.start).tz(userTimezone).format('DD.MM.YY HH:mm') }} <raw-text>-</raw-text> {{ statsMeeting.end == null ? $t('meetings.now') : $date.utc(statsMeeting.end).tz(userTimezone).format('DD.MM.YY HH:mm') }}</small>
+          </h5>
+        </template>
+        <b-alert show variant="info"><i class="fas fa-info-circle"></i> {{ $t('meetings.stats.noBreakoutSupport')}}</b-alert>
+
+        <line-chart v-if="statsMeeting" :height="250" :chart-data="chartData" :chart-options="chartOptions"></line-chart>
+      </b-modal>
+      <!-- Attendance modal -->
+      <b-modal :static="modalStatic" size="xl" hide-footer id="attendanceModal" title-tag="div" title-class="w-100">
+        <template #modal-title >
+          <div class="d-flex justify-content-between align-items-center">
+            <h5 v-if="attendanceMeeting">{{ $t('meetings.attendance.modalTitle',{room: room.name}) }}
+              <br><small>{{ $date.utc(attendanceMeeting.start).tz(userTimezone).format('DD.MM.YY HH:mm') }} <raw-text>-</raw-text> {{ $date.utc(attendanceMeeting.end).tz(userTimezone).format('DD.MM.YY HH:mm') }}</small>
```

Comment:
We should use the locale date format with [$d](https://kazupon.github.io/vue-i18n/guide/datetime.html)!